datasets/ee/international_sanctions/crawler.py

```diff
@@ -0,0 +1,70 @@
+import re
+
+from zavod import Context, helpers as h
+
+
+BELARUS_URL = (
+    "https://www.vm.ee/en/sanctions-government-republic-view-situation-belarus"
+)
+HUMAN_RIGHTS_URL = "https://www.vm.ee/subjektide-nimekiri-vabariigi-valitsuse-sanktsioon-inimoiguste-jargimise-tagamiseks"
+
+
+def crawl_item_belarus(raw_name: str, context: Context):
+
+    match = re.search(r"([^\(\n]+)\s*(?:\((.+)\))?", raw_name)
+    if match:
+        name = match.group(1)
+        aliases = match.group(2).split("; ") if match.group(2) else []
+    else:
+        context.log.warning(f"Could not parse name: {raw_name}")
+        return
+
+    entity = context.make("Person")
+    entity.id = context.make_id(name)
+    entity.add("name", name)
+    entity.add("topics", "sanction")
+
+    for alias in aliases:
+        entity.add("alias", alias)
+
+    context.emit(entity, target=True)
+
+
+def crawl_item_human_rights(raw_name: str, context: Context):
+
+    match = re.search(r"\d+\.\s*([^(\n]+)(?:\s*\(also\s*([^)]+)\))?", raw_name)
+    if match:
+        name = match.group(1).strip()
+        aliases = match.group(2).split("; ") if match.group(2) else []
+
+    last_name, first_name = name.split(", ")
+
+    entity = context.make("Person")
+    entity.id = context.make_id(name)
+    h.apply_name(entity, first_name=first_name, last_name=last_name)
+    entity.add("topics", "sanction")
+
+    for alias in aliases:
+        entity.add("alias", alias)
+
```

Comment:
Can we make a sanctions object in each of these that emits the title of the list in some form, and the source URL? Perhaps we should also include a `sourceUrl` with the entities themselves.